### PR TITLE
Fixes delete button for downloaded games from homebrew store in UWP

### PR DESCRIPTION
**Pull request by Antx64**
This PR fixes the delete function for homebrew store downloads. Also, had to add condition to void Register_sceUsbMic() function. It was causing errors on build for UWP.

### CODE REVIEW

File: Core/HLE/sceUsbMic.h [hunk 1]
```diff
@@ -16,5 +16,7 @@
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
 #pragma once
-
+//UWP doesn't like this for some reason
```
Comment (Florin9doi) [inline Core/HLE/sceUsbMic.h]: sceUsbMic.c/h should be added to CoreUWP/CoreUWP.vcxproj and CoreUWP/CoreUWP.vcxproj.filters

Comment (hrydgard) [reply Core/HLE/sceUsbMic.h]: I just did that on master, so preferably just rebase this pull request on master.

**Comment by hrydgard** [comment]
Good catch. Please rebase on master and remove the sceUsbMic checks, and I'll merge.

**Comment by Antx64** [comment]
Just did the rebase and git pushed. Thanks

**Pull request merged**